graders/r/Dockerfile

```diff
@@ -1,17 +1,19 @@
 # Borrow the leg work done by the rocker project
+# c.f. https://github.com/rocker-org/rocker-versioned/tree/master/tidyverse
 FROM rocker/tidyverse:3.6.1
 
 # Set a new label for the image
 LABEL org.label-schema.license="GPL-2.0" \
       org.label-schema.vcs-url="https://github.com/PrairieLearn/PrairieLearn" \
       org.label-schema.vendor="PrairieLearn" \
-      maintainer="James Balamuta <balamut2@illinois.edu>"
+      maintainer="James Joseph Balamuta <balamut2@illinois.edu>"
 
-# Add a script to install any additional R packages
+# Add a script to install any additional R packages from CRAN or GitHub
 COPY r-requirements.R /
 
 # Modify the image to meet with PrairieLearn specifications
-RUN useradd ag `# Enable autograde user for PrairieLearn` \                        
-    && su root -c "Rscript /r-requirements.R" `# Install the additional packages`
+# Enable autograde user for PrairieLearn and create home directory
+RUN adduser --disabled-password --gecos "" ag \                        
```

Comment:
The alternative here would be to use:

```bash
useradd -m ag
```

I switched it over to `adduser` as I think this is the preferred approach.